src/Features/Core/Portable/Snippets/CommonSnippetIdentifiers.cs

```diff
@@ -13,5 +13,6 @@ internal static class CommonSnippetIdentifiers
     public const string ConsoleWriteLine = "cw";
     public const string Constructor = "ctor";
     public const string Property = "prop";
+    public const string RequiredProperty = "propr";
```

Comment:
"Required proeprty" is a C#-specific snippet (same as `propi` for instance), so please move this to `CSharpSnippetIdentifiers`

Reply:
Good catch, moved it out